dwds/debug_extension/tool/copy_builder.dart

```diff
@@ -7,25 +7,39 @@ import 'package:build/build.dart';
 /// Factory for the build script.
 Builder copyBuilder(_) => _CopyBuilder();
 
-/// Copies the [_backgroundJsId] file to [_backgroundJsCopyId].
 class _CopyBuilder extends Builder {
   @override
   Map<String, List<String>> get buildExtensions => {
-        _backgroundJsId.path: [_backgroundJsCopyId.path]
+        "web/{{}}.dart.js": ["build/web_prod/{{}}.js"],
```

Comment:
Do I understand correctly that you are outputting into `<package root>/build` here, but that also you run the build with `-o build`?

Do you get a mix of "to source" written output, and then the merged output all in the same directory?

Would it work to change this builder to be to-cache instead of to-source, and make the directory move into a subdirectory of `web/`? 

```suggestion
        "web/{{}}.dart.js": ["web/web_prod/{{}}.js"],
```

I _think_ that should result in the `-o build` copying these files into where you expect them.

Reply:
Ah thank you! Updated so now:
- building with DDC builds to a `dev_build` directory
- building with dart2js builds to the `build` directory, but then copies just the `build/web` directory over to `prod_build`